### DISCUSSION TITLE
I would like to ask if there is a plan to follow the official website?

### DISCUSSION THREAD

**Discussion by lzm0x219** [solidjs/solid#384] [Q&A] [answered]
I think it is a bit inconvenient to read the documentation on github.

**Comment by ryansolid** [comment, marked as answer]
Yeah, it's been in the works for a while. Just lacking finishing the content. I keep intending to get to it but I haven't found a break from development for months. I will be changing gears soon.

**Reply by lzm0x219** [reply]
thanks~ bro.😄
